parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -58,17 +58,17 @@ configureK8sCustomCloud() {
         KUBERNETES_FILE_DIR=$(dirname "${AZURE_JSON_PATH}")
         K8S_CLIENT_CERT_PATH="${KUBERNETES_FILE_DIR}/k8s_auth_certificate.pfx"
         echo $SERVICE_PRINCIPAL_CLIENT_SECRET_CERT | base64 --decode > $K8S_CLIENT_CERT_PATH
-        # shellcheck disable=SC2002
-        cat "${AZURE_JSON_PATH}" | \
+        # shellcheck disable=SC2002,SC2005
+        echo $(cat "${AZURE_JSON_PATH}" | \
```

Comment:
Is this an unrelated bug fix?

Reply:
Because I changed this in #1230 (and tested the commands individually), but if so: sorry!

Reply:
@mboersma  It is ok , Because Azure Stack does not have the pipeline in the PR , we are working on that. hopefully we can get the it done in next month.

yes, it is a bug.